frontend/src/api/k8s/nimAccounts.ts

```diff
@@ -0,0 +1,11 @@
+import { k8sListResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { NIMAccountModel } from '~/api/models';
+import { NIMAccountKind } from '~/k8sTypes';
+
+export const listAccounts = async (namespace: string): Promise<NIMAccountKind[]> =>
```

Comment:
Minor nitpick: rename to `listNIMAccounts`

everything else looks good though to me

Reply:
Updated